Makefile

```diff
@@ -18,7 +18,11 @@ bin/python: config.yml
 	bin/pip install --upgrade pip
 	bin/pip install --upgrade setuptools
 
-install: bin/python bin/elastic-ingest
+bin/pip-licenses: bin/python
+	bin/pip install pip-licenses
```

Comment:
This should go to our requirements files. Out of current requirements files best one is `tests.txt`, but we might wanna already rename it to `dev.txt` or something similar, cause we also have our linters there

Reply:
Let's keep it the way you've implemented it, good enough for now definitely